src/Controllers/MovieController.cs

```diff
@@ -24,7 +24,7 @@ private IUnitOfWork UnitOfWork
             get { return this._unitOfWork; }
         }
 
-        [HttpGet("")]
+        [HttpGet(""), Authorize]
```

Comment:
The controller is already requesting for authorization, do we need this method to also request for it?

Reply:
Incorrect rebase